config/initializers/devise_token_auth.rb

```diff
@@ -0,0 +1,14 @@
+DeviseTokenAuth.setup do |config|
```

Comment:
set  max_number clients allowed to avoid millions of tokens per user making it slow to log in. 
Also check out for other configurations